ignite/metrics/epoch_metric.py

```diff
@@ -60,7 +72,8 @@ def update(self, output):
             try:
                 self.compute_fn(self._predictions, self._targets)
             except Exception as e:
-                raise RuntimeError("Problem with `compute_fn`:\n {}".format(e))
+                warnings.warn("Probably, there can be a problem with `compute_fn`:\n {}".format(e),
```

Comment:
What's the reason for switching to a warning here?

Reply:
Replace raise RuntimeException by warning in EpochMetric, as this test can potentially stops the computation: if the input data is such that a single batch does not satisfy the condition required by compute_fn but complete data satisfies all conditions.

Reply:
Got it. Thanks for clarifying!